tests/iterationIndex.json

```diff
@@ -1,7 +1,11 @@
 {
     "_meta": {
         "topic": "air_quality",
-        "key": "id"
+        "key": "id",
+        "proto": {
```

Comment:
Let’s not change existing tests, but rather add a new test

Reply:
I am adding a `interationIndex-proto.json` file to use instead